boards/intel/adsp/intel_adsp_ace30_ptl_sim.dts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 Intel Corporation
```

Comment:
I don't see this as a moved file, usually copyrights are for when the file is introduced.